jujugui/views.py

```diff
@@ -102,6 +102,7 @@ def config(request):
         # The external services' URLs.
         'charmstoreURL': settings['jujugui.charmstore_url'],
         'jemURL': settings['jujugui.jem_url'],
+        'jimmURL': settings['jujugui.jimm_url'],
```

Comment:
As mentioned before I'll do a jem removal branch. 

There will definitely be tests added.